### PR TITLE
BPF docs - Add a note about IPVS and MKE docs to 3.28

### PR DESCRIPTION
**Pull request by sridhartigera**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-enterprise/operations/ebpf/enabling-ebpf.mdx [hunk 1]
```diff
@@ -161,6 +161,11 @@ To re-enable it:
 ```bash
 kubectl patch networks.operator.openshift.io cluster --type merge -p '{"spec":{"deployKubeProxy": true}}'
 ```
+:::note
+
+If you are running kube-proxy in IPVS mode, switch to IPTables mode before disabling.
```
Comment (tomastigera) [inline calico-enterprise/operations/ebpf/enabling-ebpf.mdx]: yes, switch IPTable to iptables

Comment (tomastigera) [reply calico-enterprise/operations/ebpf/enabling-ebpf.mdx]: should it be part of `### Configure kube-proxy` ?

**Comment by ctauchen** [comment]
@sridhartigera Can you provide some context here? Links to tickets, say?

**Comment by sridhartigera** [comment]
> @sridhartigera Can you provide some context here? Links to tickets, say?

@ctauchen we don't support migrating from IPVS mode to eBPF dataplane. We have seen few users trying this. Hence it will be better to call this out explicitly.

File: calico-enterprise/operations/ebpf/enabling-ebpf.mdx (hunk 1 above)
Comment (sridhartigera) [reply calico-enterprise/operations/ebpf/enabling-ebpf.mdx]: Its part of configure kube-proxy section.

File: calico-enterprise/operations/ebpf/install.mdx [hunk 2]
```diff
@@ -42,6 +42,7 @@ and in particular, pushing the networking capabilities of the latest Linux kerne
 **Unsupported platforms**
 
 - GKE
+- MKE
```
Comment (ctauchen) [inline calico-enterprise/operations/ebpf/install.mdx]: If we're removing support for MKE, do we not need to remove from other areas? See Line 170 in `enabling-ebpf.mdx`. I can't make any sense of this. If there's a qualification, this must be made explicit.

Comment (sridhartigera) [reply calico-enterprise/operations/ebpf/install.mdx]: Sorry. That was a mistake. we are not removing support for MKE. Reverted ^^

**Pull request merged**